src/app.module.ts

```diff
@@ -0,0 +1,29 @@
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { ConfigModule } from '@nestjs/config';
+import { LoggerModule } from 'nestjs-rollbar';
+
+import * as databaseConfig from './common/database-connection';
+import { ConversationModule } from './conversation/conversation.module';
+import { UserMessageModule } from './userMessage/userMessage.module';
+import { MessageModule } from './message/message.module';
+
+@Module({
+  controllers: [],
+  providers: [],
+  imports: [
+    ConfigModule.forRoot({ isGlobal: true }),
+    TypeOrmModule.forRoot(databaseConfig),
+    ConversationModule,
+    UserMessageModule,
+    MessageModule,
+    LoggerModule.forRoot({
+      accessToken: process.env.ROLLBAR_TOKEN,
+      environment: process.env.NODE_ENV,
```

Comment:
This should probably be a separate ENV var that provides the _application environment_ as opposed to the Node environment? (i.e. should be `preview` or `staging` or `production`, or `review-12345` if a review app).

Reply:
Updated